frontend/src/components/CampaignPlanning.vue

```diff
@@ -11,6 +11,10 @@
       <img :src="'static/loading' + getRandomInt(3) + '.gif'" style="width: 120px; height: 120px;"/>
       <h3>Loading table...</h3>
     </div>
+    <div class="align-center mb-4">
```

Comment:
Hm, maybe it would be better to also add an if to show these buttons only if any of the stacks are not empty? I mean
```
v-if="undoStack.length || redoStack.length"
```
I think 90% of the people opening that page will have no intention in editing it, so we could hide the Undo and Redo buttons for them. What do you think?